src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
+      return this.result + num;
+    },
+    subtract: function (num) {
+      return this.result - num;
+    },
+    multiply: function (num) {
+      return this.result * num;
+    },
+    divide: function (num) {
+      return this.result / num;
+    },
+    reset: function (num) {
```

Comment:
[NAMING] - The 'reset' method should not take any parameters according to the task description. Remove the unused 'num' parameter.